mailpoet/lib/Cron/DaemonActionSchedulerRunner.php

```diff
@@ -28,6 +28,13 @@ class DaemonActionSchedulerRunner {
   /** @var ActionScheduler */
   private $actionScheduler;
 
+  /**
+   * The inital value is set based on default cron execution limit battle tested in MailPoet custom cron runner (an older version of the background processing).
+   * The default limit in PHP is 30s so it leaves 10 execution margin.
```

Comment:
I find this long explanation quite unnecessary. I think saying "default is 20 seconds" is enough.